packages/lexical-playground/src/plugins/CommentPlugin/index.tsx

```diff
@@ -930,6 +904,43 @@ export default function CommentPlugin({
           }
         });
       }),
+      // style text nodes with comment IDs
+      editor.registerUpdateListener((payload) => {
+        const {mutatedNodes} = payload;
+        editor.getEditorState().read(
+          () => {
+            if (mutatedNodes) {
```

Comment:
You know how much I love an early return
```suggestion
            if (!mutatedNodes) {
              return;
            }
```